Extension/src/LanguageServer/extension.ts

```diff
@@ -873,16 +873,20 @@ function handleMacCrashFileRead(err: NodeJS.ErrnoException | undefined | null, d
 
     // Get rid of the process names on each line and just add it to the start.
     const process1: string = "cpptools-srv";
-    const process2: string = "cpptools";
+    const process2: string = "cpptools-wordexp";
+    const process3: string = "cpptools";
     if (data.includes(process1)) {
         data = data.replace(new RegExp(process1 + "\\s+", "g"), "");
         data = `${process1}\t${binaryVersion}\n${data}`;
     } else if (data.includes(process2)) {
         data = data.replace(new RegExp(process2 + "\\s+", "g"), "");
         data = `${process2}\t${binaryVersion}\n${data}`;
+    } else if (data.includes(process3)) {
```

Comment:
Can this be turned into a helper function to reuse repeating code?

```

if (intputData.includes(inputProcess)) {
        intputData = intputData.replace(new RegExp(inputProcess+ "\\s+", "g"), "");
        intputData= `${inputProcess}\t${binaryVersion}\n${data}`;
```

Reply:
Yeah, it didn't seem worth the effort, but I could do that if you want.

Reply:
It's only saving two lines, so either way. If it grows to 4+ processes, then maybe worth it for a helper function.